### DISCUSSION TITLE
Avoid restoring window size if "almost maximize"

### DISCUSSION THREAD

**Discussion by garrucho** [rxhanson/Rectangle#874] [Ideas]
Not sure if it makes sense for more people, but changing window position after an "almost maximize" doesn't represents an _unsap_ for me.

My workflow is to _almost maximize_ just to apply a default size that isn't my full screen (QHD, too much canvas), and then sometimes I'll be moving them to different positions, but at this moment, they'll be restored to the previous size.

At the same time, I would like to keep the windows being restored after a real unsnapping because it makes all the sense to me – so I didn't disable the option.

**Comment by rxhanson** [comment]
Thanks for the feedback. This makes sense to me, but I think it would have to be added in as a Terminal command config. Introducing an inconsistency would likely introduce a lot of false bug reports. 

I don't mind adding this in, but it might take a while to get to it. A PR would definitely be welcome if anyone wants this in sooner.
